docusaurus-search-local/src/server/utils/processDocInfos.ts

```diff
@@ -16,76 +17,108 @@ export function processDocInfos(
     blogRouteBasePath,
     ignoreFiles,
   }: ProcessedPluginOptions
-): DocInfoWithFilePath[] {
-  return routesPaths
-    .map<DocInfoWithRoute | undefined>((url: string) => {
-      // istanbul ignore next
-      if (!url.startsWith(baseUrl)) {
-        throw new Error(
-          `The route must start with the baseUrl "${baseUrl}", but was "${url}". This is a bug, please report it.`
-        );
+): VersionDocInfo[] {
+  const emptySet = new Set();
+  let versionData: any = [{ versionOutDir: outDir, docs: emptySet }];
+  if (plugins) {
+    const docsPluginData = plugins.find(element => element.name === "docusaurus-plugin-content-docs");
+    if (docsPluginData) {
+      versionData = [];
+      // @ts-ignore
+      const loadedVersions = docsPluginData.content.loadedVersions;
```

Comment:
The type of content here is unknown and I'm not experienced enough with typescript to know if there's another better way to solve this. The type of loadedVersions is exposed as LoadedVersion[] so maybe there's a nicer way to at least use that?